packages/tailwindcss/src/scss/_container.scss

```diff
@@ -0,0 +1,24 @@
+.hn {
+  &-container {
+    &-header {
```

Comment:
&-header and &-content has the same rules so we can just put it in the &-container class 
`
.hn-container {
  @apply grid;
  @apply grid-cols-4 sm:grid-cols-8 md:grid-cols-12 lg:grid-cols-12 xl:grid-cols-12;
  @apply gap-5 sm:gap-4 lg:gap-6 xl:gap-7;
  @apply mx-5 sm:mx-4 lg:mx-6 xl:mx-auto;

  &-header {
    @apply xl:w-[var(--container-max-width-header)];
  }

  &-body {
    @apply xl:w-[var(--container-max-width-body)];
  }
`